wavedump/src/dark_vs_laser.py

```diff
@@ -0,0 +1,73 @@
+import csv
+import matplotlib.pyplot as plt
+import numpy as np
+
+try:
+    with open('Fit_Data.csv', 'r') as file:
```

Comment:
I think it would be good to have an argument to pass in the name here, you can set the default to "Fit_Data.csv"